docs/user/Alerts.md

```diff
@@ -0,0 +1,64 @@
+## MachineWithoutValidNode
+Each machine should have a valid node reference and one or more machines does not after 10 minutes.
```

Comment:
i still think this reads a little odd, i think it would be clearer like this:
```suggestion
One or more machines does not have a valid node reference after 10 minutes.
```
otoh, if you want to keep the info about each machine needing a valid reference i might rewrite like this:
```suggestion
Each machine should have a valid node reference, this alert signifies that one or more machines does not after 10 minutes.
```